spring-boot/admin/src/main/resources/templates/admin/fragments/gateway_health_monitor.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
+
+
+<body>
+<div th:fragment="gateway-health-monitor">
+    <div class="x_panel">
+        <div class="x_title">
+            <h2>
+                Gateway Health Monitor
+                <a data-toggle="modal" data-target=".help-gateway-health-monitor">
+                    <i class="fa fa-question-circle"></i>
+                </a>
+            </h2>
+            <div class="clearfix"></div>
+        </div>
+        <div class="x_content">
+            <div class="demo-container" style="height:300px">
+
+                <div th:switch="${gatewayHealthMonitorProxyData.statusSummary}">
+                                    <span th:case="${T(eu.coatrack.api.ProxyHealthStatus).WARNING}">
+                                    <span th:text="#{admin.dashboard.gatewayhealthmonitor.state.warning_critical}"></span>
+                                    <span class="warning-exclamation-mark">&#9888;</span></span>
+                    <span th:case="${T(eu.coatrack.api.ProxyHealthStatus).CRITICAL}">
+                                    <span th:text="#{admin.dashboard.gatewayhealthmonitor.state.warning_critical}"></span>
+                                    <span class="critical-exclamation-mark">&#9888;</span></span>
+                    <span th:case="${T(eu.coatrack.api.ProxyHealthStatus).OK}">
+                                    <span th:text="#{admin.dashboard.gatewayhealthmonitor.state.ok}"></span>
+                                    <span class="everything-ok-symbol">&#10004;</span></span>
+                    <span th:case="${T(eu.coatrack.api.ProxyHealthStatus).NEVER_CONNECTED}">
+                                    <span th:text="#{admin.dashboard.gatewayhealthmonitor.state.notconnected}"></span>
+                                    <span class="warning-not-connected-exclamation-mark">&#9888;</span>
+                                    </span>
+                </div>
+
+                <table class="health-monitor-data-table">
+                    <tr>
+                        <th th:text="#{admin.dashboard.gatewayhealthmonitor.gatewayname}"></th>
+                        <th th:text="#{admin.dashboard.gatewayhealthmonitor.status}"></th>
+                        <th th:text="#{admin.dashboard.gatewayhealthmonitor.lastcontact}"></th>
+                        <th th:text="#{admin.dashboard.gatewayhealthmonitor.notification.switch}"></th>
+                    </tr>
+                    <tr th:each="gateway : ${gatewayHealthMonitorProxyData.gatewayDataListForGatewayHealthMonitor}">
+                        <td align="left" th:text="${gateway.name}"></td>
+
+                        <td align="center" th:switch="${gateway.status}">
+                            <span th:case="${T(eu.coatrack.api.ProxyHealthStatus).WARNING}">
+                                <span th:class="'neutral-state-symbol warning-state'"></span></span>
```

Comment:
Do we need one `span` nested inside another here? Wouldn't it be sufficient to just do this?
```
 <span th:case="${T(eu.coatrack.api.ProxyHealthStatus).WARNING}" th:class="'neutral-state-symbol warning-state'"></span>
```

Reply:
I thinkit would be sufficient so I applied the changes on the code. However, due to the problems on the config server I can´t start coatrack on my local, so I can't test it